mysql-test/lib/My/Debugger.pm

```diff
@@ -144,7 +144,7 @@ sub do_args($$$$$) {
   my $v = $debuggers{$k};
 
   # on windows mtr args are quoted (for system), otherwise not (for exec)
-  sub quote($) { $_[0] =~ /[; ]/ ? "\"$_[0]\"" : $_[0] }
+  sub quote($) { $_[0] =~ /[; ]|--replicate-rewrite-db/ ? "\"$_[0]\"" : $_[0] }
```

Comment:
No, `--replicate-rewrite-db` string doesn't need to be quoted, `>` does. So, the fix should be
```
-  sub quote($) { $_[0] =~ /[; ]/ ? "\"$_[0]\"" : $_[0] }
+  sub quote($) { $_[0] =~ /[; >]/ ? "\"$_[0]\"" : $_[0] }
```
or even better `[; ><*?$']` and may be more.
Also it'll break if the parameter itself contains double quotes. Want to fix that too?

Reply:
Thanks Serg, I have wrote alternative `A2` in PR description as your suggested fix and will update accordingly
Sure let's fix double quotes in parameter value too in the same PR.